### PR TITLE
Generate controller todo_item with CRUD

### PR DESCRIPTION
**Pull request by ziGihouse**
generate controller todo_item with actions index; show; create; edit; update; destroy

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
+  get 'todo_item/index'
+  get 'todo_item/show'
+  get 'todo_item/new'
+  get 'todo_item/create'
```
Comment (Raileanv) [inline config/routes.rb]: here should be `post`
check all todo_item related routes

File: app/controllers/todo_item_controller.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class TodoItemController < ApplicationController
+  def index
```
Comment (Raileanv) [inline app/controllers/todo_item_controller.rb]: fill all actions with appropriate code

**Comment by ziGihouse** [comment]
Odin fail necevo serios
Rest need to review

File: app/controllers/todo_item_controller.rb (hunk 2 above)
Comment (ziGihouse) [reply app/controllers/todo_item_controller.rb]: need more information

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 class ApplicationController < ActionController::Base
   before_action :authenticate_user!
-  before_action :authenticate_admin!
-  before_action :configure_permitted_parameters, if: :devise_controller?
+  # before_action :authenticate_admin!
+  # before_action :configure_permitted_parameters, if: :devise_controller?
```
Comment (Raileanv) [inline app/controllers/application_controller.rb]: let's use only user for now

File: config/routes.rb [hunk 4]
```diff
@@ -1,6 +1,18 @@
...
-  devise_for :admins
+  # get 'todo_item/index'
+  # get 'todo_item/show'
+  # get 'todo_item/new'
+  # post 'todo_item/create'
+  # get 'todo_item/edit'
+  # patch 'todo_item/update'
+  # delete 'todo_item/destroy'
+
+resources :todo_items
```
Comment (Raileanv) [inline config/routes.rb]: correct form is in plural

**Comment by Raileanv** [comment]
Have a look, and when you are ready you can merge 🇲🇩

**Pull request merged**